### PR TITLE
updated ORex data area in IsisPref

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

Kernels for ORex were updated so the default config has been changed to reflect that. 

## Related Issue

#4060 

## How Has This Been Tested?

Have not tested locally, just gonna let Jenkins do it's thing.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: CHANGELOG.md
Comment (jessemapel) [inline CHANGELOG.md:72]: This would be a good opportunity to add that we have support for the OREx Bennu data.

**Review by jessemapel — DISMISSED** [review]

File: CHANGELOG.md
Comment (scsides) [reply CHANGELOG.md:72]: The TestPreference file needs to have the same change. Changing this may cause some tests to fail.

**Review by scsides — DISMISSED** [review]

**Comment by robotprogrammer22** [comment]
It looks like a couple of tests for osirisrex are failing on jenkins at the moment. I'm not sure if it's related to the change in path. I'll take a look and see if I can figure it out.

**Comment by acpaquette** [comment]
@Kelvinrr the Orex Camera unit test is still failing on this one

File: isis/src/osirisrex/tsts/mapcam/Makefile
Comment (scsides) [inline isis/src/osirisrex/tsts/mapcam/Makefile]: Is there a reason for removing these redirects? Most every makefile test ignores stdout and stderr. Same for other tests below.

Comment (Kelvinrr) [reply isis/src/osirisrex/tsts/mapcam/Makefile]: I only did that to get the error messages coming from Jenkins, since it wasn't reproducible locally. That's how I determined the issue is the image existing pre public release and are not in the new kernels' time range. They'll be reverted when the tests are working.

**Comment by jessemapel** [comment]
What's the status on this? Do you need a new test image that is actually of Bennu for the OCams unit test?

**Comment by KrisBecker** [comment]
I would highly recommend you add at least one image each for the PolyCam and SamCam instruments. Note PolyCam has over 94 separate focus position settings each with distinctly different focal lengths, distortion parameters and center positions.

I would also recommend you add tests using the shape model (bennu_g_00880mm_alt_obj_0000n00000_v020.bds) provided with the kernel set to determine how it interacts with the ISIS system. 

Please let me know which images you select and I can provide corresponding caminfo data for comparisons. 

When do you anticipate releasing these kernels?

Thanks...

**Comment by jlaura** [comment]
@KrisBecker Those updates sound like they might be a new PR. Will have to have @Kelvinrr weigh in. If so, PRs from external contributors are always solicited! Especially so when they have expertise in the instrument.

@Kelvinrr Pining you again in case you missed the bump from @jessemapel.

**Comment by KrisBecker** [comment]
Possibly could lead to a PR but would be good to know if its needed. A few tests should inform us.

**Comment by Kelvinrr** [comment]
@jessemapel This has been hanging for some time because I haven't really been putting those support hours in. It basically needs some test images that are within the time ranges for the public kernels, as the original images were from the privated kernels. 

I agree that @KrisBecker's updates are out of scope for this PR.

**Comment by jessemapel** [comment]
I agree that adding more test coverage is outside of the scope of this, all that should be done here is updating the existing PolyCam and MapCam tests. I'll open up a new issue regarding improving the camera model test coverage that and it can be prioritized with the next mission support sprint.

**Comment by jessemapel** [comment]
@KrisBecker  What is special about that DSK that is different from the existing DSK tests? I am hesitant to add a 100 MB file to the test data requirements unless it is absolutely necessary.

**Comment by KrisBecker** [comment]
@jessemapel the new Bennu DSK is not different, per se, than other DSKs in the ISIS system. It is, however, pretty much required for good cartographic processing. Bennu has some rather large boulders, meters high, that are well modeled in the DSK.

I was just asking that it be included in a test to confirm ISIS4 can interoperate with the DSK to support mapping operations.

**Comment by jessemapel** [comment]
Okay, I agree then. Orthorectifying onto the DSK with the test image by hand and seeing if it intersects reasonably make sense to me. I can do that check after Kelvin finishes with this.

**Comment by Kelvinrr** [comment]
I think this one is good to go, after merging I can update the truth data for mapcam.

**Review by jessemapel — COMMENTED** [review]
Looks good, just need to delete the old unitTest.cpp for the OCams. The Truth file is deleted but not the unittest and it's causing a test failure.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**
